scripts/release.kts

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prefab.release
+
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileReader
+import java.io.FileWriter
+import java.util.*
```

Comment:
but not java.io.*? (i'm fine with * but i thought you avoided it. either way, seems like it would make sense to be consistent?)

Reply:
IntelliJ (I'm not doing any of these manually) for some reason special cases `java.util.*` by default. The inconsistency is weird, but I don't feel strongly enough about it to change it.